triggers/service/auth/src/main/scala/com/daml/oauth/server/Server.scala

```diff
@@ -29,6 +32,7 @@ import scala.language.postfixOps
 // request to /authorize are immediately redirected to the redirect_uri.
 class Server(config: Config) {
   private val jwtHeader = """{"alg": "HS256", "typ": "JWT"}"""
+  private val tokenLifetimeSeconds = 24 * 60 * 60
```

Comment:
I wonder if we should move this to a test directory or include `dummy`  in the package name. I keep getting confused, e.g, here by why we can chose the lifetime ourselves until I realize that is our dummy test server.